app/views/articles/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<%= link_to_unless_current "English", locale: 'en' %>
+<%= link_to_unless_current "Svenska", locale: 'sv' %>
+
+<%= link_to t('login'), "/" %>
```

Comment:
There is a login button in application layout file

Reply:
removed